frontend/src/Data/Form.ts

```diff
@@ -445,3 +422,95 @@ export const increaseObservationCountFormInit: () => Map<string, FormItem> = ()
       ],
     ]
   )
+
+
+export const updatePositionFormInit: () => Map<string, FormItem> = () =>
+  new Map(
+    [
+      ["position_id",
+        {
+          order: 1,
+          label: "Position ID",
```

Comment:
it appears that if I input a negative number for the position ID the UI freezes.

(I don't mean in any specific place, it seems to be common to all such fields)